### DISCUSSION TITLE
Link references in _Layout.cshtml are not valid for a Site created under a virtual directory

### DISCUSSION THREAD

**Discussion by robovill** [Dotnet-Boxed/Templates#109] [Q&A]
In the “_Layout.cshtml’ template I found that the most of the link references (eg. the favicon urls) are not placed within a Url.Content() call and so refer to an invalid location when the website is placed under a virtual directory of another web-site.

**Comment by RehanSaeed** [comment]
Thanks @robovill, expect a fix in the next couple of days. Willing to accept a PR for this.

**Comment by RehanSaeed** [comment]
I just remembered that I am using a `~` character at the beginning of the URL. There is a tag helper in ASP.NET Core which turns this into `Url.Content()`. Are you saying that this does not work?
